### PR TITLE
Remove some arguments of Formatter.instance.format and spec

### PR DESCRIPTION
**Pull request by akihikodaki**

### CODE REVIEW

File: app/lib/formatter.rb [hunk 1]
```diff
@@ -9,17 +9,16 @@ class Formatter
 
   include ActionView::Helpers::TextHelper
 
-  def format(status, attribute = :text, paragraphize = true)
```
Comment (ykzts) [inline app/lib/formatter.rb]: FYI: deleted with #3020.

**Review by ykzts — APPROVED** [review]

File: spec/lib/formatter_spec.rb [hunk 2]
```diff
@@ -1,193 +1,285 @@
...
-        expect(subject).to match('<a href="http://google.com/" rel="nofollow noopener" target="_blank"><span class="invisible">http://</span><span class="">google.com/</span><span class="invisible"></span></a>')
+      it 'has valid URL' do
+        is_expected.to include 'href="http://google.com/"'
       end
+    end
+
+    context 'matches a stand-alone IDN URL' do
+      let(:text) { 'https://nic.みんな/' }
 
-      it 'contains a mention' do
```
Comment (abcang) [inline spec/lib/formatter_spec.rb]: Is this test missing?

File: spec/lib/formatter_spec.rb [hunk 3]
```diff
@@ -1,193 +1,285 @@
...
 
-      it 'contains a hashtag' do
-        result = Formatter.instance.format(local_status_with_mention)
-        expect(result).to match('/tags/hashtag" class="mention hashtag" rel="tag">#<span>hashtag</span></a>')
+      it 'has display URL' do
+        is_expected.to include '<span class="">nic.みんな/</span>'
       end
     end
 
-    context 'with cashtag' do
```
Comment (abcang) [inline spec/lib/formatter_spec.rb]: Is this test missing?

File: spec/lib/formatter_spec.rb [hunk 4]
```diff
@@ -1,193 +1,285 @@
...
+      let(:status)  { Fabricate(:status, text: '<p>a text by a nerd who uses an HTML tag in text</p>', uri: nil) }
 
-      it 'has valid url' do
-        expect(subject).to eq '<p>http://www\.google\.com</p>'
+      it 'returns raw text' do
+        is_expected.to eq '<p>a text by a nerd who uses an HTML tag in text</p>'
       end
     end
 
-    context 'concatenates hashtag and URL' do
```
Comment (abcang) [inline spec/lib/formatter_spec.rb]: Is this test missing?

**Review by abcang — CHANGES_REQUESTED** [review]

File: spec/lib/formatter_spec.rb (hunk 2 above)
Comment (akihikodaki) [inline spec/lib/formatter_spec.rb]: Moved to #3540.

File: spec/lib/formatter_spec.rb (hunk 3 above)
Comment (akihikodaki) [inline spec/lib/formatter_spec.rb]: Moved to #3540.

File: spec/lib/formatter_spec.rb (hunk 4 above)
Comment (akihikodaki) [inline spec/lib/formatter_spec.rb]: Moved to #3540.

File: spec/lib/formatter_spec.rb (hunk 2 above)
Comment (akihikodaki) [inline spec/lib/formatter_spec.rb]: Some fragments are still at https://github.com/tootsuite/mastodon/pull/3541/files/4d54599f00864b00df090d11c0a0c26cdf2b8142#diff-b36adf141aa6e4040f0bd5c46ee60fd0R154 or somewhere else.

**Comment by akihikodaki** [comment]
It is following #3540 up and should not be merged before it. I forgot to note that.

**Review by abcang — APPROVED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**